composer.json

```diff
@@ -4,16 +4,16 @@
     "description": "Seamless integration of Algolia Search into your Symfony project.",
     "license": "MIT",
     "require": {
-        "php": "^5.6 || ^7.0",
-        "symfony/framework-bundle": "~2.5|~3.0",
-        "symfony/finder": "~2.5|~3.0",
-        "symfony/property-access": "~2.5|~3.0",
+        "php": ">=5.6",
```

Comment:
Unless you know that the bundle will work with PHP 8, you shouldn't use the `>=` operator. The constraint to `^7.0` is there to specifically avoid running into a version containing a BC break.